Documentation/TestingFramework.rst

```diff
@@ -30,10 +30,9 @@ version-dependent code switches might be necessary. The upside is that there
 is only one branch to maintain, which makes adding new features (and all other
 code changes) a lot less of a hassle.
 
-The `Nimut testing framework <https://github.com/Nimut/testing-framework>`__
```

Comment:
We then can drop this document (and the reference to it) altogether.

Reply:
Hi,
I wanted to clarify your comment about dropping the document and the reference to it. Are you suggesting that we remove it entirely because it is no longer relevant, or do you think we should retain it for future reference? Your insight would be appreciated.
Thanks

Reply:
Yes, please let’s drop the file altogether.

Reply:
Thank you for the clarification. I have removed the document and its reference as requirement. Please review and let me know if any further changes are needed.